tap_stripe/__init__.py

```diff
@@ -1034,24 +1067,13 @@ def sync():
     # Loop over streams in catalog
     for catalog_entry in Context.catalog['streams']:
         stream_name = catalog_entry['tap_stream_id']
-        # Sync records for stream
         if Context.is_selected(stream_name):
-            if not Context.is_sub_stream(stream_name):  # Run the sync for parent-streams
-                sync_stream(stream_name)
-                # This prevents us from retrieving 'events.events'
+            # Run the sync for only parent streams/only child streams/both parent-child streams
```

Comment:
@namrata270998 this comment looks incorrect with the condition you have written.
Shouldn't it be "Run the sync for only parent stream/only child streams"

Reply:
No in case when both parent and child are selected, also we need to run the sync for only parent, hence this comment